bin/python-noenv.sh

```diff
@@ -1,2 +1,2 @@
 #!/bin/bash
-env -i /usr/bin/python "$@"
+env -i PBS_DEFAULT=$PBS_DEFAULT /usr/bin/python "$@"
```

Comment:
you can only pass a single argument to `python-noenv` via shebang, not two (that's just how shebang works)

and you could make it recognise something like `PASS_ENV_VAR=PBS_DEFAULT,USER,HOME` as 1st argument, and act accordingly?